inst/create_cm_template.R

```diff
@@ -0,0 +1,230 @@
+# Name: CM domain
+#
+# Label: R program to create CM Domain
+#
+# Input raw data:
+# study_sdtm_spec
+# study_controlled_terminology
+# study_raw_datasets
+#
+#
+
+library(sdtm.oak)
+library(dplyr)
+
+# Read Specification
+
+# sdtm_spec <- read_sdtm_spec(filename = "~/study/study_sdtm_spec.csv")
+
+study_ct <- read_study_ct(filename = "~/study/study_sdtm_ct.csv")
```

Comment:
should we have a template ct file along with the package? Just to get a feeling of the expected input

Reply:
I fully support your suggestion of including a template file with the package. In the realm of clinical trials, particularly non-commercial ones, we often encounter a vast array of data formats. A template, serving as a standard reference for expected inputs, could greatly assist users in structuring their data appropriately for the {sdtm.oak} package.

I understand that this is a future consideration and not necessary for the current PR.